libs/@guardian/eslint-config/package.json

```diff
@@ -10,18 +10,18 @@
 	},
 	"dependencies": {
 		"@eslint-community/eslint-plugin-eslint-comments": "4.4.1",
-		"@eslint/js": "9.19.0",
+		"@eslint/js": "9.20.0",
```

Comment:
Should this be updated independently of eslint? Which is now on 9.19.0 @oliverabrahams